record/mpp.go

```diff
@@ -0,0 +1,43 @@
+package record
+
+import (
+	"io"
+
+	"github.com/lightningnetwork/lnd/tlv"
+)
+
+type MPP struct {
+	TotalMsat   uint64
```

Comment:
`Total lnwire.MilliSatoshi`

Reply:
agreed, i have these changes locally just need to rebase everything